topics/proteomics/tutorials/neoantigen-1a-fusion-database-generation/faqs/index.md

```diff
@@ -0,0 +1,7 @@
+---
+redirect_from:
+  - /topics/proteomics/tutorials/neoantigen-1-fusion-database-generation/faqs/index.html
+redirect_to:
```

Comment:
you don't need the `redirect_to`. This file will already be served at this location, so we only need to say that the location above (in `redirect_from`) should also lead here.